src/tinybox.vue

```diff
@@ -202,6 +202,38 @@ export default {
         window.removeEventListener('keyup', this.keyup);
       }
     },
+    /*
+     * Center the thumbnails' scrollbar to the clicked image
+     */
+    index(index) {
+      let elements;
+      let el;
+      let thumbs;
+
+      if (!this.noThumbs) {
+        this.$nextTick(() => {
+          if (typeof index !== 'undefined' && index !== null) {
```

Comment:
`index` has a default value of `null`, so it's always defined -- no need to change it here :)

```suggestion
          if (index !== null) {
```

Reply:
What I also would do is to merge this `if` statement with one on line 213, so that the checking of `index` is not delayed until the next tick.